### DISCUSSION TITLE
Misorientation angle profile along a linear path ???!!!  (Is it possible ??)

### DISCUSSION THREAD

**Discussion by Alireza25101989** [mtex-toolbox/mtex#1269] [Ask Anything] [answered]
Hi

Is it feasible to measure the **Misorientation angle** changes along a **linear path** with an specific length using MTEX ???!!
For example, to draw the variation of misorientation angles versus distance from the grain boundaries ??

Regards,
Alireza

**Comment by kilir** [comment, marked as answer]
Hi,
see here: https://mtex-toolbox.github.io/EBSDProfile.html
Rüdiger

**Reply by Alireza25101989** [reply]
Much appreciated.
Thanks

Alireza
